bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/DataItemGroup.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.openthermgateway.internal;
+
+import java.util.HashMap;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * @author Arjen Korevaar - Initial contribution
+ */
+@NonNullByDefault
+public class DataItemGroup {
+    private int id;
+    private DataItem[] dataItems;
+
+    public int getID() {
+        return id;
+    }
+
+    public void setID(int id) {
+        this.id = id;
+    }
+
+    public DataItem[] getDataItems() {
+        return dataItems;
+    }
+
+    public void setDataItem(DataItem[] dataItems) {
```

Comment:
It looks like the fields and methods in the class are not used?

Reply:
It's a datastructure that represents the OpenTherm protocol specifications of data-id's, types and mapping of payload bytes. I created it to have a complete dataset of all the possible messages, but (after some refactoring) ended up not using everything.. Then again, it's very nice to be able to lookup the individual message types and relate them to the data-id's from the OT specification..

Reply:
After taking another look.. there were indeed some attribute on DataItemGroup that were now nolonger needed. I also removed some setters from DataItem, since it's intention is to be a reference list of OpenTherm commands, and therefor not to be modified once created.